timemachine/fe/utils.py

```diff
@@ -460,9 +460,9 @@ def sanitize_energies(full_us, lamb_idx, cutoff=10000):
     return np.where(abs_us < cutoff, full_us, np.inf)
 
 
-def read_sdf(fname: Union[str, Path]) -> List[Chem.Mol]:
+def read_sdf(fname: Union[str, Path], removeHs=False) -> List[Chem.Mol]:
     """Read list of mols from an SDF (without discarding hydrogens!)"""
-    supplier = Chem.SDMolSupplier(str(fname), removeHs=False)
+    supplier = Chem.SDMolSupplier(str(fname), removeHs=removeHs)
```

Comment:
hah, code change invalidates docstring --> remove docstring

Reply:
Fixed docstring in https://github.com/proteneer/timemachine/pull/1254/commits/c4f70085c4cc5b664b454302fbd44df5215131b5